core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActorTimeoutSpec.scala

```diff
@@ -60,7 +60,8 @@ class SubmissionMonitorActorTimeoutSpec(_system: ActorSystem)
 
   behavior of "SubmissionMonitorActor query timeouts"
 
-  it should "should enforce on saveEntities()" in withDefaultTestDatabase { dataSource: SlickDataSource =>
+  // TODO CORE-515: should we add a timeout in CompactEntityProvider?
+  it should "should enforce on saveEntities()" ignore withDefaultTestDatabase { dataSource: SlickDataSource =>
```

Comment:
this test fails because we no longer have a SQL timeout for this query. Do we need to add that back in?

Reply:
What happens without the timeout?  Is it hopefully unneeded because quicksilver is faster?

Reply:
Without the timeout, the query runs as long as it needs to - which hopefully with Quicksilver is faster.

Reply:
If we no longer have a timeout, is this test still relevant?

Reply:
I'd say we delete it